assets/styles/style.css

```diff
@@ -0,0 +1,27 @@
+*{
```

Comment:
Please add a space between selector and open block like so:

```suggestion
* {
```